backend/src/email/services/custom-domain.service.ts

```diff
@@ -0,0 +1,113 @@
+import dns from 'dns'
+import AWS from 'aws-sdk'
+
+import { EmailFromAddress } from '@email/models'
+
+const ses = new AWS.SES({ region: 'eu-central-1' })
```

Comment:
region should not be hardcoded - it can be parsed from SES_HOST

Reply:
Thanks for catch this; forgot about it